### PR TITLE
Add IssueEvents stream

### PR DESCRIPTION
**Pull request by ericboucher**

### CODE REVIEW

File: tap_github/streams.py [hunk 1]
```diff
@@ -412,3 +401,61 @@ def post_process(self, row: dict, context: Optional[dict] = None) -> dict:
...
+        """
+        if context and context.get("events", None) == 0:
+            self.logger.debug(f"No events detected. Skipping '{self.name}' sync.")
+            return []
+
+        return super().get_records(context)
+
+    def post_process(self, row: dict, context: Optional[dict] = None) -> dict:
+        row["issue_number"] = int(row["issue"]["number"])
+        row["issue_url"] = row["issue"]["url"]
```
Comment (Ry-DS) [inline tap_github/streams.py:434]: I don't know too much about the ELT process, but Laurent hinted that the output table should be a pure dump of the underlying schema. This looks like just cleanup to make the tap easier to use but just wondering what line we can draw for what is considered extracting vs transforming. This looks good for now tho!

File: tap_github/streams.py [hunk 2]
```diff
@@ -372,17 +372,6 @@ def get_records(self, context: Optional[dict] = None) -> Iterable[Dict[str, Any]
...
-        self, context: Optional[dict], next_page_token: Optional[Any]
-    ) -> Dict[str, Any]:
-        """Return a dictionary of values to be used in URL parameterization."""
-        params = super().get_url_params(context, next_page_token)
-        if self.replication_key:
-            since = self.get_starting_timestamp(context)
-            if since:
-                params["since"] = since
-        return params
-
```
Comment (Ry-DS) [inline tap_github/streams.py:385]: Why was this removal done?

**Review by Ry-DS — APPROVED** [review]
Looks good! Small comments not really around the code but are moreso points to discuss.

File: tap_github/streams.py (hunk 2 above)
Comment (ericboucher) [reply tap_github/streams.py:385]: I followed the feedback @laurentS has received on the parent repo.

**Comment by ericboucher** [comment]
Merged on parent repo.

**Pull request closed without merging**